src/name_bazaar/ui/components/offering/list_item.cljs

```diff
@@ -132,7 +133,7 @@
      [offering-header-missing-ownership-tag
       {:offering offering}])
 
-   (when show-sold?
+   (when (and show-sold? (not (:offering/unregistered? offering)))
```

Comment:
Please put `(not (:offering/unregistered? offering))` condition inside `offering-header-sold-tag`